### PR TITLE
removing workaround PR #1517

### PR DESCRIPTION
**Pull request by a7vicky**
### What type of PR is this?
_cleanup_

### What this PR does / why we need it?
This PR is to remove workaround of patching Validatingwebhookconfiguration deployed in Hosted cluster in hypershift mode. Currently this has been handle by PKO here : https://github.com/openshift/managed-cluster-validating-webhooks/pull/212

### CODE REVIEW

File: hack/00-osd-managed-cluster-config-production.yaml.tmpl [hunk 1]
```diff
@@ -22646,161 +22646,6 @@ objects:
...
-      matchExpressions:
-      - key: ext-hypershift.openshift.io/cluster-type
-        operator: In
-        values:
-        - service-cluster
-      - key: api.openshift.com/fedramp
-        operator: NotIn
-        values:
-        - 'true'
-    resourceApplyMode: Sync
```
Comment (rendhalver) [inline hack/00-osd-managed-cluster-config-production.yaml.tmpl:22670]: Will th

**Comment by mrbarge** [comment]
/hold cancel
This can go out now, as the required MCVW changes have been promoted.

**Pull request merged**